### ISSUE TITLE
Provide built release files

### ISSUE THREAD

**Issue by sebastienros** [nghttp2/nghttp2#1505]
I'd like to be able to download a binary from the release tab, or at least build one myself. Right now it requires external packages to be installed on the machine. The README mentions it should  be available but couldn't find it:

> The nghttp2 project regularly releases tar archives which includes nghttp2 source code, and generated build files

**Comment by StianOvrevage** [comment]
Yes, please. It's 2021! Publish some RPMs, DEBs and Docker images already :)

**Comment by StianOvrevage** [comment]
After failing to install via sources bash conveniently suggested I try

```sudo apt install nghttp2-client```

Which seems to work :)
